tests/ModelParser/Model/WithImports.php

```diff
@@ -23,4 +24,9 @@ class WithImports
      * @var Nested
      */
     private $aliasSameNamespace;
+
+    /**
+     * @var Nested[]|Collection
```

Comment:
```suggestion
     * @var Nested[]|Collection<Nested>
```

Reply:
could we test if just `Collection<Nested>` also works correctly, detecting the Nested as element type?

Reply:
Adjusted via a95fd9aec4fb795ac183a9760b8464324ac4e400